adb_shell/handle/usb_handle.py

```diff
@@ -172,7 +169,7 @@ def __init__(self, device, setting, usb_info=None, timeout_s=None):
         self._write_endpoint = None
 
         self._usb_info = usb_info or ''
-        self._timeout_s = timeout_s if timeout_s else DEFAULT_TIMEOUT_S
+        self._default_timeout_s = default_timeout_s if default_timeout_s else DEFAULT_TIMEOUT_S
```

Comment:
Is it OK for the default timeout to be 0? If so, we should check `... if default_timeout_s is not None else...`

Reply:
Yes.
Looking at the implementation of `bulkRead` and `bulkWrite` in `usb1.USBDeviceHandle`, it seems that 0 is the actual default value. 
I'll push another commit with your suggestion.